### ISSUE TITLE
Failed to download: fetch failed

### ISSUE THREAD

**Issue by lvyuemeng** [flix/flix#7831]
OS: windows 11 / WSL2 ubuntu 24.04
the problem occurs when I install flix by the extension.

**Comment by magnus-madsen** [comment]
Let us know if upgrading VSCode helps

**Comment by lvyuemeng** [comment]
Here I turned back to retry downloading flix in windows.

`Failed to download: The "readableStream" argument must be an instance of ReadableStream. Received an instance of h`

**Comment by lvyuemeng** [comment]
Strangely, I now successfully download flix.
But unfortunately, the problem of repl still exist.(It seems a general problem related to java)

**Comment by magnus-madsen** [comment]
Does the repl work if you rum flix outside of vscode?

**Comment by lvyuemeng** [comment]
> Does the repl work if you rum flix outside of vscode?

Yes, I try `java -jar $path to flix.jar` and the repl still has the problem.
before I mentioned that ammonite of scala has the same problem, so I guess it's related to java itself.

**Comment by magnus-madsen** [comment]
> > Does the repl work if you rum flix outside of vscode?
> 
> Yes, I try `java -jar $(path to flix.jar)` and the repl still has the problem. before I mentioned that ammonite of scala has the same problem, so I guess it's related to java itself.

Sorry to hear that... Indeed it sounds like a JLine or Java problem. Did you try the new Windows Terminal?

But all that said, I don't experience the same issue on my Windows machine so there must be something specific. Maybe WSL or something else?

**Comment by lvyuemeng** [comment]
> > > Does the repl work if you rum flix outside of vscode?
> > 
> > 
> > Yes, I try `java -jar $(path to flix.jar)` and the repl still has the problem. before I mentioned that ammonite of scala has the same problem, so I guess it's related to java itself.
> 
> Sorry to hear that... Indeed it sounds like a JLine or Java problem. Did you try the new Windows Terminal?
> 
> But all that said, I don't experience the same issue on my Windows machine so there must be something specific. Maybe WSL or something else?

Yes, I use the powershell5, powershell, terminal etc. I guess it's not the main factor here. WSL is fine, but Windows is not.

**Comment by lvyuemeng** [comment]
Ok, in new version of java, it works fine. And I can download flix successfully. I will close the issue soon!
